app/src/main/java/org/mozilla/fenix/settings/wallpaper/Extensions.kt

```diff
@@ -25,3 +28,37 @@ fun List<Wallpaper>.groupByDisplayableCollection(): Map<Wallpaper.Collection, Li
 }.toMap().takeIf {
     it.isNotEmpty()
 } ?: mapOf(Wallpaper.DefaultCollection to listOf(Wallpaper.Default))
+
+/**
+ * The ideal scenario is to have 3 seasonal wallpapers and 2 classic, minding the order.
+ * But in case there are less than 3 or no seasonal wallpapers at all, they are replaced by classic.
+ * Edge cases where there are less then 6 items overall would return less items in the right order.
+ */
+fun List<Wallpaper>.groupForWallpaperOnboarding(): List<Wallpaper> {
+    val result = mutableListOf(Wallpaper.Default)
+
+    val classicWallpapers = mutableListOf<Wallpaper>()
+    val seasonalWallpapers = mutableListOf<Wallpaper>()
+    for (wallpaper in this) {
```

Comment:
```suggestion
    val result = mutableListOf(Wallpaper.Default)
    val classicWallpapers = mutableListOf<Wallpaper>()
    val seasonalWallpapers = mutableListOf<Wallpaper>()
    
    for (wallpaper in this) {
```
Generally, we want a new line before and after a `for` statement block to separate out and make it easier to read. Otherwise, all our code will just be squished together with no clear logical separation from a reader's perspective.